api_tests/e2e/rfc003/btc_eth/happy.ts

```diff
@@ -27,9 +27,13 @@ setTimeout(function() {
         it("bitcoin ether", async function() {
             await alice.sendRequest(AssetKind.Bitcoin, AssetKind.Ether);
             await bob.accept();
+
             await alice.fund();
+
             await bob.fund();
+
             await alice.redeem();
+
             await bob.redeem();
```

Comment:
Can we not have these white spaces please, at least not in between same actions i.e., `alice.redeem` and `bob.redeem`.

Reply:
Why not? :)

Reply:
It doesn't make it any clearer and it uses up vertical space.  Its personal preference though :)